test/test_manifest.json

```diff
@@ -1967,5 +1967,11 @@
        "rounds": 1,
        "type": "eq",
        "about": "Free image obtained from www.unsplash.com"
+    },
+    {  "id": "issue5567.pdf",
```

Comment:
Nit: You don't really need ".pdf" in the `id`, since this is just the unique name of the test.